src/VirtualClient/VirtualClient.Monitors.UnitTests/AmdSmiQueryGpuParserUnitTests.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace VirtualClient.Monitors
+{
+    using System.Collections.Generic;
+    using System.Diagnostics;
+    using System.IO;
+    using System.Linq;
+    using System.Reflection;
+    using System.Text;
+    using System.Threading.Tasks;
+    using NUnit.Framework;
+    using VirtualClient.Common;
+    using VirtualClient.Contracts;
+
+    [TestFixture]
+    [Category("Unit")]
+    public class AmdSmiQueryGpuParserUnitTests
+    {
+        [Test]
+        public void AmdSmiQueryGpuParserParsesMetricsCorrectly()

```

Comment:
In the example I see only 1 GPU, would you like to add tests for when you have multiple GPUs?

Reply:
Sure, I didn't consider this use case and I can add a test for that.